content/manuals/engine/network/ca-certs.md

```diff
@@ -0,0 +1,130 @@
+4. Drag-and-drop the downloaded certificate into the list of certificates. Enter your password if prompted.
+5. Find the newly added certificate, double-click it, and expand the **Trust** section.
+6. Set **Always Trust** for the certificate. Enter your password if prompted.
+7. Start Docker Desktop and verify that `docker pull` works, assuming Docker Desktop is configured to use the MITM proxy.
+
+### Windows
+
+1. Download the CA certificate for your MITM proxy software.
+2. Open your web browser, go to **Settings** and open **Manage certificates**
+3. Select the **Trusted Root Certification Authorities** tab.
+4. Select **Import**, then browse for the downloaded CA certificate.
+5. Select **Open**, then choose **Place all certificates in the following store**.
+6. Ensure **Trusted Root Certification Authorities** is selected and select **Next**.
+7. Select **Finish** and then **Close**.
+8. Start Docker Desktop and verify that `docker pull` works, assuming Docker Desktop is configured to use the MITM proxy.
+
+## Add CA certificates to a running container
+
+> [!NOTE]
+> The following commands are for an Ubuntu-based container. If your container
+> uses a different Linux distribution, use equivalent commands for package
+> management (`apt-get`, `update-ca-certificates`, and so on).
+
+To add a CA certificate to a running Linux container:
+
+1. Download the CA certificate for your MITM proxy software.
+2. If the certificate is in a format other than `.crt`, convert it to `.crt` format:
+
+   ```console {title="Example command"}
+   $ openssl x509 -in cacert.der -inform DER -out myca.crt
+   ```
+
+3. Copy the certificate into the running container:
+
+    ```console
+    $ docker cp myca.crt <containerid>:/tmp
+    ```
+
+4. Attach to the container:
+
+    ```console
+    $ docker exec -it <containerid> sh
+    ```
+
+5. Ensure the `ca-certificates` package is installed (required for updating certificates):
+
+    ```console
+    # apt-get update && apt-get install -y ca-certificates
+    ```
+
+6. Copy the certificate to the correct location for CA certificates:
+
+    ```console
+    # cp /tmp/myca.crt /usr/local/share/ca-certificates/root_cert.crt
+    ```
+
+7. Update the CA certificates:
+
+    ```console
+    # update-ca-certificates
+    ```
+
+    ```plaintext {title="Example output"}
+    Updating certificates in /etc/ssl/certs...
+    rehash: warning: skipping ca-certificates.crt, it does not contain exactly one certificate or CRL
+    1 added, 0 removed; done.
+    ```
+
+8. Verify that the container can communicate via the MITM proxy:
+
+    ```console
+    # curl https://google.com
+    ```
+
+    ```plaintext {title="Example output"}
+    <html><head><meta http-equiv="content-type" content="text/html;charset=utf-8">
+    <title>301 Moved</title></head><body>
+    <h1>301 Moved</h1>
+    The document has moved
+    <a href="https://www.google.com/">here</a>.
+    </body></html>
+    ```
+
+## Add CA certificates at build-time
```

Comment:
Minor comment: I would expect the build-time steps to be more useful since you only need to configure it once, rather than after every container create. Perhaps re-order the sections to put build nearer the top? Maybe we could say that the runtime steps are useful for debugging and understanding what's happening?

Reply:
Sure, I can do that!

Reply:
updated https://github.com/docker/docs/compare/7a101948b4941d3c52e1c89de879c9c39dbaa2c2..ae9a5c35849ca76ffa21a002458fa47f9b655f86